### ISSUE TITLE
Package dan200.computercraft.api.client is missed from common-api package

### ISSUE THREAD

**Issue by SirEdvin** [SquidDev-CC/CC-Tweaked#1706]
### Minecraft Version

1.20.x

### Version

1.109.4

### Details

So, it seems that package dan200.computercraft.api.client  is missed from common-api and because of this it is not possible to register TurtleUpgradeModeller at all ...

![зображення](https://github.com/cc-tweaked/CC-Tweaked/assets/3713151/8bbd62d9-b543-433d-9ef5-774f64267e72)

**Comment by SquidDev** [comment]
Oh, this is so odd. All the other published jars are fine, this one is just missing the classes entirely!

The task dependencies are definitely there, so I have no clue why this is happened!

```text
$ ./gradlew taskTree :common:jar

:common:jar
+--- :common:classes
... 
+--- :common:clientClasses
```

I think my only option here is just to publish a new version - will try to do that in a sec. Though I'm not especially enthusiastic about the fact that the build scripts are so unreliable.

**Comment by SquidDev** [comment]
Have published 1.109.5 to fix this. Knowing my luck, there'll probably be some other issue with the Forge jar now :).
